src/figma/tokens.json

```
@@ -559,231 +559,231 @@
           "type": "color"
         }
       },
       "primary": {
         "default": {
-          "value": "hsla(205, 98%, 53%, 1)",
+          "value": "#1098FC",
           "description": "(blue400: #1098FC) primary.default should be used for all primary action components such as buttons or links.",
           "type": "color"
         },
         "alternative": {
-          "value": "hsla(205, 97%, 63%, 1)",
+          "value": "#43AEFC",
           "description": "(blue300: #43AEFC) primary.alternative should be used as an alternative to primary.default for things such as hover states",
           "type": "color"
         },
         "muted": {
-          "value": "hsla(205, 98%, 53%, .15)",
+          "value": "#1098FC10",
           "description": "(blue400: #1098FC 10% opacity) primary.muted is a very low contrasting primary variant for things such as alert backgrounds. primary.muted and primary.inverse should not be used together in a foreground and background combination",
           "type": "color"
         },
         "inverse": {
-          "value": "hsla(0, 0%, 99%, 1)",
+          "value": "#FCFCFC",
           "description": "(white010: #FCFCFC) primary.inverse should be used only as the foreground element on top of primary.default and primary.alternative. It is intended to be the most contrasting color to primary.default. It should meet all AA and AAA accessibility standards such as the text or icon of a primary button.",
           "type": "color"
         },
         "disabled": {
-          "value": "hsla(205, 98%, 53%, .5)",
+          "value": "#1098FC50",
           "description": "(blue400: #1098FC) 50% opacity) primary.disabled should be used for all disabled primary action components such as buttons or links.",
           "type": "color"
         }
       },
       "secondary": {
         "default": {
-          "value": "hsla(24, 93%, 60%, 1)",
+          "value": "##F8883",
           "description": "(orange400: ##F8883B) secondary.default should be used for any secondary actions. It should not be used for any negative connotations such as warnings or errors as it is quite closely tied to the MetaMask Fox",
           "type": "color"
         },
         "alternative": {
-          "value": "hsla(25, 93%, 70%, 1)",
+          "value": "#FAA66C",
           "description": "(orange300: #FAA66C) secondary.alternative should be used as an alternative to secondary.default for things such as hover states",
           "type": "color"
         },
         "muted": {
-          "value": "hsla(24, 93%, 60%, .15)",
+          "value": "##F8883B10",
           "description": "(orange400: ##F8883B 10% opacity) secondary.muted is a very low contrasting secondary variant for things such as alert backgrounds. secondary.muted and secondary.inverse should not be used together in a foreground and background combination",
```

Has an extra hash and should be 15% see other comment.

```suggestion
          "value": "#F8883B15",
          "description": "(orange400: #F8883B 15% opacity) secondary.muted is a very low contrasting secondary variant for things such as alert backgrounds. secondary.muted and secondary.inverse should not be used together in a foreground and background combination",
```